program_1.rb

```diff
@@ -0,0 +1,14 @@
+while 1
```

Comment:
do it with class
we follow object oriented pattern
write a class and  methods. create object and do things inside it.